Solopov/DZ5_AjaxAsynchronousMethodAndCSS/ServletTestList/src/main/java/ui/Account/LoginServlet.java

```diff
@@ -0,0 +1,40 @@
+package ui.Account;
+
+import services.UserService;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import java.io.IOException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class LoginServlet extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+
+
+        HttpSession session=req.getSession();
+        String login,password;
+        login=req.getParameter("uname");
+        password=req.getParameter("psw");
+        Logger.getLogger(login+" "+password);
```

Comment:
What level of logging is it?

Reply:
Something messed up. Must be level: info
And getLog() should refer to the declaration of an object of type Logger